src/components/PastCommitNode.tsx

```diff
@@ -47,22 +47,12 @@ export class PastCommitNode extends React.Component<
     };
   }
   getBranchesForCommit() {
-    const idAbrev = this.props.pastCommit.commit.slice(0, 7);
```

Comment:
It seems like some older previous commits are also included in this PR, like for this pr: https://github.com/jupyterlab/jupyterlab-git/pull/280/

This isn't ideal because it makes the diff a bit longer and includes unrelated changes.